src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -493,13 +495,11 @@ AthensSurfaceExamples class >> example9 [
 	paint := surf createFormPaint: f.
 	
 	surf drawDuring: [:can |
-
 		can 
 			setPaint: paint;
-			drawShape: (0@0 corner: 100@200).
-		
-		Display getCanvas drawImage: surf asForm  at: 0@0 ]
+			drawShape: (0@0 corner: 100@200)].
 
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?